lib/stdlib/doc/src/orddict.xml

```diff
@@ -81,6 +99,15 @@
           An exception is generated if the initial value associated with
           <c><anno>Key</anno></c> is not a list of values.</p>
         <p>See also section <seeerl marker="#notes">Notes</seeerl>.</p>
+          <p><em>Example:</em></p>
```

Comment:
This example can be improved in the same way as the `orddict:append/3` example. That is, show an actual append operation on an existing entry, and start out with a single existing key so that the example becomes clear without a call to `orddict:fetch/2`.

Reply:
Thanks for the feedback. I followed your suggestion